src/devices/LiquidLevelSwitch/LiquidLevelSwitchBase.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.LiquidLevelSwitch
+{
+    /// <summary>
+    /// Represents base class for Liquid Level Switches
+    /// </summary>
+    public abstract class LiquidLevelSwitchBase : IDisposable
```

Comment:
I think `Base` suffix is not needed here since name is already generic enough without it

Reply:
Should I remove the specific implementation Llc200d3sh and change the sample to use the LiquidLevelSwitch?

Reply:
Should I remove the specific implementation Llc200d3sh and change the sample to use the LiquidLevelSwitch?

Reply:
No preference on removing but regardless I think LiquidLevelSwitch should be showcased in the sample (so both or just generic implementation)